src/server/server_family.cc

```diff
@@ -2598,9 +2598,6 @@ void ServerFamily::ReplConf(CmdArgList args, ConnectionContext* cntx) {
       VLOG(2) << "Received client ACK=" << ack;
       cntx->replication_flow->last_acked_lsn = ack;
       return;
-    } else if (cmd == "ACL-CHECK") {
```

Comment:
This breaks replication when replica is of lower version. My two cents is that we don't care for this case since this setup doesn't really make much sense. Let me know if you disagree

Reply:
please keep the code in master and add a TODO with your anniversary date to remove this code.

Reply:
I will add an alarm on my calendar as well :wink: